cookbooks/aws-parallelcluster-platform/resources/enroot/partial/_enroot_rhel.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+#
+# Copyright:: 2013-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+action :install_package do
+  package prerequisites do
+    retries 3
+    retry_delay 5
+  end
+
+  bash "Install enroot" do
+    user 'root'
+    cwd node['cluster']['sources_dir']
+    code <<-ENROOT_INSTALL
+    set -e
```

Comment:
A good thumb rule is to maintain Indentation when we add a code block. linters should take care of it though!